collect_app/src/main/java/org/odk/collect/android/logic/FormController.java

```diff
@@ -133,7 +133,7 @@ public static synchronized void initializeJavaRosa(IPropertyManager mgr) {
 
     public FormController(File mediaFolder, FormEntryController fec, File instancePath) {
         this.mediaFolder = mediaFolder;
-        formEntryController = fec;
+        this.formEntryController = fec;
```

Comment:
Hi. Glad to see new tests. Did you add `.this` for consistency? I suggest it’s better without—that higher signal to noise trumps consistency in this case.